src/components/HighlightedText/index.js

```diff
@@ -47,6 +46,8 @@ HighlightedText.propTypes = {
      * the text not highlighted
      */
     textComponent: PropTypes.elementType,
+    /** A boolean that when it is true wraps the text in a span tag, and when it is false wraps it in a p tag, this lets you display the text block or inline */
```

Comment:
I think the description should not mention internal details but what it finally do